luigi/server.py

```diff
@@ -45,6 +45,7 @@
 import time
 
 import pkg_resources
+from prometheus_client import CONTENT_TYPE_LATEST
```

Comment:
This breaks because we are missing `prometheus_client` as a dependency in `setup.py`. So instead of pulling the dependency, we can hardcode the string.

Reply:
Or better if we could somehow let contrib module set additional headers...